dev_tools/tests/test_nxdl_utils.py

```diff
@@ -48,11 +49,144 @@ def test_get_node_at_nxdl_path():
     )
     assert node.attrib["name"] == "long_name"
 
+    nxdl_file_path = os.path.join(
```

Comment:
Can you refactor use (throughout) of `os.path` to use [`pathlib.Path()`](https://docs.python.org/3/library/pathlib.html) objects?  Examples are in other modules here, such as [`nxdl_utils.py`](https://github.com/nexusformat/definitions/blob/979ec79105cae92d256a544802218a86c6dc709c/dev_tools/utils/nxdl_utils.py#L46).
These objects make the code easier to write, read, and review.

Reply:
Sure, I changed it in this file.